autoprotocol/builders.py

```diff
@@ -1833,6 +1917,7 @@ def __init__(self):
         self.blowdown_params = ["gas", "blow_rate", "vortex_speed"]
 
     def get_mode_params(self, mode, mode_params):
+        #pylint: disable=R0912
```

Comment:
why are we changing formatting on code not included in this PR? If you want to reformat code not in scope of this PR, please open a new PR

Reply:
same as above... didn't realize tox ignores R/C codes. reverting!